### PR TITLE
Updated warning in readiness probes

### PR DESCRIPTION
**Pull request by AmarNathChary**
Updated the warning in readiness probes as suggested.

### CODE REVIEW

**Comment by AmarNathChary** [comment]
This PR fixes: #43709

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -358,8 +358,8 @@ Readiness probes runs on the container during its whole lifecycle.
 {{< /note >}}
 
 {{< caution >}}
-Liveness probes *do not* wait for readiness probes to succeed.
-If you want to wait before executing a liveness probe you should use
+The readiness probe does not wait for the liveness probe to succeed.
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:361]: Maybe add something like "The readiness probe and the liveness probe do not have dependencies on each other" would be better.

**Review by kumarankit999 — COMMENTED** [review]
Looks good to me here!

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 1 above)
Comment (AmarNathChary) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: Hi @tengqm Changes made as per the suggestion PTAL!

**Review by nate-double-u — COMMENTED** [review]
Thanks for the update @AmarNathChary. I've made some suggestions inline:

**Review by kumarankit999 — COMMENTED** [review]
You can change as suggested by @nate-double-u

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 2]
```diff
@@ -358,8 +358,8 @@ Readiness probes runs on the container during its whole lifecycle.
 {{< /note >}}
 
 {{< caution >}}
-Liveness probes *do not* wait for readiness probes to succeed.
-If you want to wait before executing a liveness probe you should use
+The readiness probe and the liveness probe do not have dependencies on each other to succeed.
```
[the remark this answers is not in the document]
Comment (AmarNathChary) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:361]: The suggestions look good. I will make those changes.

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 3]
```diff
@@ -358,8 +358,8 @@ Readiness probes runs on the container during its whole lifecycle.
...
+If you want to wait before executing a "readiness" probe, you should use
```
[the remark this answers is not in the document]
Comment (AmarNathChary) [reply content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: Hi @nate-double-u changes were made as per the suggestion PTAL!

**Comment by AmarNathChary** [comment]
@tengqm can you PTAL? at this PR.

**Comment by tengqm** [comment]
/lgtm
/approve
Thanks.

**Pull request merged**